### PR TITLE
feat: add the examples of contract sync-deploy and async-deploy

### PR DESCRIPTION
**Pull request by sulenn**

### CODE REVIEW

**Comment by fisco-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- examples/kvtable_test_async/main.go  8
- examples/KVTableTest.go  10
- examples/kvtable_test_sync/main.go  8
         

Clones added
============
- examples/kvtable_test_async/main.go  5
- examples/KVTableTest.go  13
- examples/kvtable_test_sync/main.go  5
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=6082680&bid=19912795)

File: examples/README.md [hunk 1]
```diff
@@ -76,3 +76,120 @@ AMOP（Advanced Messages Onchain Protocol）即链上信使协议，旨在为联
...
+-   准备需要编译的智能合约
+-   配置好相应版本的solc编译器
+-   构建go-sdk的合约编译工具abigen
+-   编译生成go文件
+-   准备建立ssl连接需要的证书
+-   使用生成的go文件进行合约部署、调用
+
+详细可参考：[合约开发样例](https://fisco-bcos-documentation.readthedocs.io/zh_CN/latest/docs/sdk/go_sdk/contractExamples.html)
+
+## 合约开发案例
```
Comment (fisco-dev) [inline examples/README.md:155]: Codacy found an issue: [[no-duplicate-headings] Do not use headings with similar content (142:1)](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=6082680)

File: examples/README.md [hunk 2]
```diff
@@ -76,3 +76,120 @@ AMOP（Advanced Messages Onchain Protocol）即链上信使协议，旨在为联
...
+    
+    publish topic success
+    2020/08/27 16:02:37 publish message: Hi, FISCO BCOS! 0 
+    2020/08/27 16:02:39 publish message: Hi, FISCO BCOS! 1 
+    2020/08/27 16:02:41 publish message: Hi, FISCO BCOS! 2 
+    2020/08/27 16:02:43 publish message: Hi, FISCO BCOS! 3 
+    2020/08/27 16:02:45 publish message: Hi, FISCO BCOS! 4
+    ```
+
+# 合约开发案例
```
Comment (fisco-dev) [inline examples/README.md:142]: Codacy found an issue: [[no-multiple-toplevel-headings] Don’t use multiple top level headings (1:1)](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=6082680)

**Comment by fisco-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- examples/kvtable_test_async/main.go  8
- examples/KVTableTest.go  10
- examples/kvtable_test_sync/main.go  8
         

Clones added
============
- examples/kvtable_test_async/main.go  5
- examples/KVTableTest.go  13
- examples/kvtable_test_sync/main.go  5
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=6082680&bid=19912795)

File: examples/README.md (hunk 2 above)
Comment (fisco-dev) [inline examples/README.md:142]: Codacy found an issue: [[no-multiple-toplevel-headings] Don’t use multiple top level headings (1:1)](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=6082680)

**Comment by fisco-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- examples/kvtable_test_sync/main.go  8
- precompiled/crud/crud_service_test.go  3
- examples/kvtable_test_async/main.go  8
- examples/KVTableTest.go  10
         

Clones added
============
- examples/kvtable_test_sync/main.go  5
- precompiled/crud/crud_service_test.go  5
- precompiled/crud/crud_service.go  6
- precompiled/crud/table_factory.go  2
- examples/kvtable_test_async/main.go  5
- precompiled/crud/crud.go  6
- examples/KVTableTest.go  13
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=6082680&bid=19912795)

File: examples/README.md (hunk 2 above)
Comment (fisco-dev) [inline examples/README.md:142]: Codacy found an issue: [[no-multiple-toplevel-headings] Don’t use multiple top level headings (1:1)](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=6082680)

**Comment by fisco-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- examples/kvtable_test_sync/main.go  8
- examples/kvtable_test_async/main.go  8
- precompiled/crud/crud_service_test.go  4
- examples/KVTableTest.go  10
         

Clones added
============
- examples/kvtable_test_sync/main.go  5
- examples/kvtable_test_async/main.go  5
- precompiled/crud/crud_service.go  6
- precompiled/crud/crud.go  6
- precompiled/crud/crud_service_test.go  12
- precompiled/crud/table_factory.go  2
- examples/KVTableTest.go  13
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=6082680&bid=19912795)

File: examples/README.md (hunk 2 above)
Comment (fisco-dev) [inline examples/README.md:142]: Codacy found an issue: [[no-multiple-toplevel-headings] Don’t use multiple top level headings (1:1)](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=6082680)

**Review by bxq2011hust — APPROVED** [review]

**Comment by fisco-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- examples/KVTableTest.go  10
- precompiled/crud/crud_service_test.go  4
- examples/kvtable_test_async/main.go  8
- examples/kvtable_test_sync/main.go  8
         

Clones added
============
- precompiled/crud/crud.go  6
- precompiled/crud/crud_service.go  6
- precompiled/crud/table_factory.go  2
- examples/KVTableTest.go  13
- precompiled/crud/crud_service_test.go  12
- examples/kvtable_test_async/main.go  5
- examples/kvtable_test_sync/main.go  5
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=6082680&bid=19912795)

**Pull request merged**